lib/Model/Config/AppConfig.hs

```diff
@@ -50,6 +50,8 @@ data AppConfigMail = AppConfigMail
   , _appConfigMailName :: String
   , _appConfigMailEmail :: String
   , _appConfigMailHost :: String
+  , _appConfigMailPort :: Maybe Int
+  , _appConfigMailUseSSL :: Bool
```

Comment:
Same as the comment above

Reply:
Solved in https://github.com/ds-wizard/dsw-server/pull/42/commits/3b85b9807c60716a599351ead0ff5d672863e81e